pkg/kapis/devops/v1alpha3/scm/scmhandler.go

```diff
@@ -29,11 +29,15 @@ import (
 // handler holds all the API handlers of SCM
 type handler struct {
 	k8sClient client.Client
+	client.Client
```

Comment:
IMO, `k8sClient client.Client` and `client.Client` are the same type. How about removing one of them?